lib/AST/ASTVerifier.cpp

```diff
@@ -2651,7 +2651,7 @@ class Verifier : public ASTWalker {
     }
 
     void verifyChecked(VarDecl *var) {
-      if (!var->hasInterfaceType())
+      if (!var->hasInterfaceType() || var->isInvalid())
```

Comment:
I'm worried that this is too broad and I'm going to narrow it down to ObjC properties only.